proposals/lip-0053.md

```diff
@@ -88,7 +87,7 @@ Setup to gather the required mainchain information:
 Create a cross-chain update transaction for a given height `h1`:
 
 * Find a [signed certificate][lip-0061] in the mainchain block headers for a height (say `h2`) higher or equal to `h1`. This will be used as the `certificate` property of the transaction.
-* The property `inboxUpdate.crossChainMessages` lists all CCMs that have been included in the sidechain outbox up to `h2` (and which have not been included on the sidechain yet). In this case, the `messageWitness` will be empty.
+* The property `inboxUpdate.crossChainMessages` lists all CCMs that have been included in the sidechain outbox up to `h2` (and which have not been included on the sidechain yet). In this case, the `messageWitnessHashes` will be empty.
```

Comment:
This is not required a subset is also possible.

Reply:
changed